src/Raven.Server/Documents/Handlers/Processors/Attachments/AttachmentHandlerProcessorForBulkPostAttachment.cs

```diff
@@ -102,19 +102,26 @@ protected override async ValueTask GetAttachmentsAsync(DocumentsOperationContext
                 // Handle exceptions
                 foreach (var streamTask in tasks)
                 {
-                    try
+                    _ = streamTask.ContinueWith(t =>
```

Comment:
I think you *have* to wait here.
Because otherwise, aren't you still keeping it running after cancel?